docs/using.rst

```diff
@@ -41,6 +41,13 @@ which image to install to which slot, which scripts to execute etc.
 bundles must always have a ``.raucb`` file name suffix in order to ensure that
 RAUC treats them as bundles.
 
+While the ``--cert`` and ``--key`` argument are mandatory for signing and must
+provide the certificate and private key that should be used for creating the
+signature, the ``--keyring`` argument is optional and (if given) will be used
+for verifying the trust chain validity of the created signature after creation.
```

Comment:
The 'created' before signature is redundant.

Reply:
removed